.changeset/perfect-doors-switch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/icons': major
```

Comment:
Adding icons is an additive change, not a destructive one, and thus counts as a minor change. [Learn more about semantic versioning on the official website](https://semver.org/).

```suggestion
'@sumup/icons': minor
```